### DISCUSSION TITLE
Expression database for other species than human?

### DISCUSSION THREAD

**Discussion by eSchulz7** [Open-Systems-Pharmacology/Forum#864] [Questions & Problems]
Hello,

is there an expression database for other species than human? I can only find the human gene expression database under Open-Systems-Pharmacology/ Suite Downloads.

regards,

Elena Schulz

**Comment by StephanSchaller** [comment]
There are a couple of databases in preparation but not yet released. Maybe the involved parties can comment about release plans.

**Comment by eSchulz7** [comment]
Is it possible to find and integrate a database for antoher species myself or to manually change the data?
